packages/scripts/src/findFaq.ts

```diff
@@ -0,0 +1,139 @@
+import { strict as assert } from "assert";
+import { Db, ObjectId } from "mongodb";
+
+import {
+  Conversation,
+  SomeMessage,
+  AssistantMessage,
+  FunctionMessage,
+} from "mongodb-chatbot-server";
+import { clusterize, DbscanOptions } from "./clusterize";
+import { findCentroid } from "./findCentroid";
+
+export type QuestionAndResponses = {
+  embedding: number[];
+  question: string;
```

Comment:
might it be useful to keep the whole `UserMessage` object? 

that way you'd have the original `content`, `preprocessedContent` and `contentForLlm`.

Reply:
Def